mlflow/pyspark/ml/__init__.py

```diff
@@ -775,6 +776,8 @@ def autolog(
     :param silent: If ``True``, suppress all event logs and warnings from MLflow during pyspark ML
                    autologging. If ``False``, show all events and warnings during pyspark ML
                    autologging.
+    :param log_eval_metrics: If ``True``, evaluation metrics are logged. Defaults to ``True``.
+                             See the post training metrics section for details.
```

Comment:
Can we use the term `post training metrics` rather than `evaluation` metrics and clarify that these are logged when users call metric APIs such as `Evaluator.evaluate()`?

Reply:
```suggestion
    :param log_post_training_metrics: If ``True``, post training metrics are logged. Defaults to
                                      ``True``. See the `post training metrics`_ for more details.

# `post training metrics`_ is rendered as a link
```

Can we just insert a link to the post training section?

Reply:
sure!

Reply:
@dbczumar Could you take a look at this page? https://10907-136202695-gh.circle-artifacts.com/0/docs/build/html/python_api/mlflow.sklearn.html#post-training-metrics